cmds/core/du/du.go

```diff
@@ -86,9 +86,12 @@ func (c *cmd) run() error {
 }
 
 func (c *cmd) du(file string) (int64, error) {
-	var blocks int64
+	var (
+		blocks int64
+		err    error
```

Comment:
why are you declaring error here? you can use the err := form at line 94.

Reply:
good point